### ISSUE TITLE
Bugs in brapi_get_field() when field is of type array.

### ISSUE THREAD

**Issue by ekcannon** [tripal/brapi#15]
This may be a PHP version issue. Running PHP 7.1, this type of statement:
`$identifier = $object->$field_fetcher['object_key'];`
fails with: 
`Array to string conversion` 
and 
`Undefined property: stdClass::$Array`
Even though PHP admits that `$field_fetcher['object_key']` is of type `string`.

This fixes the problem:
```
$ok = $field_fetcher['object_key'];
$identifier = $object->$ok
```
But is rather ugly. Hoping there's a nicer solution.

**Comment by guignonv** [comment]
Could you give a try to this syntax:
$identifier = $object->{$field_fetcher['object_key']};

**Comment by ekcannon** [comment]
Yes! And much nicer than my hack. I ran into this same problem a while back and had forgotten that I solved it then with {}'s.

**Comment by guignonv** [comment]
Should be solved by commit 40e3160b42fc45975c3764e8b1471a8d7e70856f
